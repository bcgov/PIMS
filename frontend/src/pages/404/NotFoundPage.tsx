import React from 'react';
import { Container, Row, Col } from 'react-bootstrap';
import { Link } from 'react-router-dom';

export const NotFoundPage = () => {
  return (
    <Container fluid={true}>
      <Row>
        <Col>
          <h1>Page not found</h1>
          <Link to="/mapview">Go back to the map</Link>
        </Col>
      </Row>
    </Container>
  );
};
