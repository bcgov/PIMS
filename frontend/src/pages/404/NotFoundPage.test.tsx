import React from 'react';
import renderer from 'react-test-renderer';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import { NotFoundPage } from './NotFoundPage';

const history = createMemoryHistory();

describe('NotFoundPage', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Router history={history}>
          <NotFoundPage />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
