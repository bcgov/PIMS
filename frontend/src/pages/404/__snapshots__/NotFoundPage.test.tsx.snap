// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <h1>
        Page not found
      </h1>
      <a
        href="/mapview"
        onClick={[Function]}
      >
        Go back to the map
      </a>
    </div>
  </div>
</div>
`;
